templates/repo/issue/card.tmpl

```diff
@@ -0,0 +1,66 @@
+{{with .Issue}}
+	{{if eq $.Page.Project.CardType 1}}{{/* Images and Text*/}}
+		<div class="card-attachment-images">
+			{{range (index $.Page.issuesAttachmentMap .ID)}}
+				<img src="{{.DownloadURL}}" alt="{{.Name}}" />
+			{{end}}
+		</div>
+	{{end}}
+	<div class="content gt-p-0">
+		<div class="gt-df gt-items-start">
+			<div class="issue-card-icon">
+				{{template "shared/issueicon" .}}
+			</div>
+			<a class="issue-card-title muted issue-title" href="{{.Link}}">{{.Title | RenderEmoji ctx | RenderCodeBlock}}</a>
+			{{if and $.isPinned $.Page.isRepoAdmin}}
+				<a role="button" class="issue-card-unpin muted gt-df gt-ac" data-tooltip-content={{ctx.Locale.Tr "repo.issues.unpin_issue"}} data-issue-id="{{.ID}}" data-unpin-url="{{$.Page.Link}}/unpin/{{.Index}}">
+					{{svg "octicon-x" 16}}
+				</a>
+			{{end}}
+		</div>
+		<div class="meta gt-my-2">
+			<span class="text light grey muted-links">
+				{{if not $.Page.Repository}}{{.Repo.FullName}}{{end}}#{{.Index}}
+				{{$timeStr := TimeSinceUnix .GetLastEventTimestamp ctx.Locale}}
+				{{if .OriginalAuthor}}
+					{{ctx.Locale.Tr .GetLastEventLabelFake $timeStr (.OriginalAuthor|Escape) | Safe}}
+				{{else if gt .Poster.ID 0}}
+					{{ctx.Locale.Tr .GetLastEventLabel $timeStr (.Poster.HomeLink|Escape) (.Poster.GetDisplayName | Escape) | Safe}}
+				{{else}}
+					{{ctx.Locale.Tr .GetLastEventLabelFake $timeStr (.Poster.GetDisplayName | Escape) | Safe}}
+				{{end}}
+			</span>
+		</div>
+		{{- if .MilestoneID}}
+		<div class="meta gt-my-2">
+			<a class="milestone" href="{{$.Page.RepoLink}}/milestone/{{.MilestoneID}}">
+				{{svg "octicon-milestone" 16 "gt-mr-2 gt-vm"}}
+				<span class="gt-vm">{{.Milestone.Name}}</span>
+			</a>
+		</div>
+		{{- end}}
+		{{if $.Page.LinkedPRs}}
+		{{- range index $.Page.LinkedPRs .ID}}
+		<div class="meta gt-my-2">
+			<a href="{{$.Page.RepoLink}}/pulls/{{.Index}}">
+				<span class="gt-m-0 text {{if .PullRequest.HasMerged}}purple{{else if .IsClosed}}red{{else}}green{{end}}">{{svg "octicon-git-merge" 16 "gt-mr-2 gt-vm"}}</span>
+				<span class="gt-vm">{{.Title}} <span class="text light grey">#{{.Index}}</span></span>
+			</a>
+		</div>
+		{{- end}}
+		{{end}}
+	</div>
+
+	{{if or .Labels .Assignees}}
+	<div class="extra content labels-list gt-p-0 gt-pt-2">
+		{{range .Labels}}
+			<a target="_blank" href="{{$.Page.RepoLink}}/issues?labels={{.ID}}">{{RenderLabel ctx .}}</a>
+		{{end}}
+		<div class="right floated">
+			{{range .Assignees}}
+				<a target="_blank" href="{{.HomeLink}}" data-tooltip-content="{{ctx.Locale.Tr "repo.projects.column.assigned_to"}} {{.Name}}">{{ctx.AvatarUtils.Avatar . 28 "mini gt-mr-3"}}</a>
```

Comment:
Isn't there a template for that?
`templates/shared/user/avatarlink.tmpl` or something like that?